src/app/controllers/events_controller.rb

```diff
@@ -1,8 +1,19 @@
 class EventsController < ApplicationController
   def show
-    @event = Event.includes(:sessions).find(params[:id])
+    @event = if params[:id] == 'current'
+      Event.current_event
```

Comment:
Does this also need `.includes(:sessions)`? I'm not really sure.

Reply:
IIRC there's a huge SQL query storm if it's not there. Could try making the request and checking the logs to be sure….

Reply:
This was a good catch: it turns out _neither_ branch needs `.includes(:sessions)`.

(The JSON form is a hot query mess, but I'm not sure anybody’s actually using it, so I left the mess hot.)